src/com_digicom/site/models/cart.php

```diff
@@ -972,7 +972,10 @@ function updateOrder($order_id, $result, $data, $pay_plugin, $status, $items, $c
 		$type = 'process_order';
 		if($status != 'Refund')
 		{
-			$orderTable->amount_paid = $orderTable->amount_paid + $data['total_paid_amt'];
+			if((int)$data['total_paid_amt']>0){
+				$orderTable->amount_paid = (int)$orderTable->amount_paid + (int)$data['total_paid_amt'];
```

Comment:
price value could be float number. so don't force it to be int

Reply:
Oh! that's my bad. It was really unexpected. Thank you so much and let me fix this.